integration_tests/test_dataset_add.py

```diff
@@ -218,6 +218,34 @@ def test_dataset_add_not_eo3(index: Index, ls8_eo3_product, eo3_wo_dataset_doc)
     assert isinstance(_err, BadMatch)
 
 
+def test_dataset_opt_measurements(
+    index: Index, s1_dataset_doc, ga_s1_full_product
+) -> None:
+    doc, uri = s1_dataset_doc
+    doc["product"]["name"] = ga_s1_full_product.name
+
+    doc2ds = Doc2Dataset(index)
+    _ds, _err = doc2ds(doc, uri)
```

Comment:
I think something has changed in Ruff 0.13, but it now considers a leading underscore in the variable name to signal that the variable isn't used (which is consistent with ESLint).

So rename `_ds` to `ds` since that is used, and just put a `_` instead of `_err` for the unused component of the tuple. (And same fix for the other used/unused variables below.)